### PR TITLE
[APB-2143][MP, PT]: Implemented ContinueUrl for redirect to invitations

### PR DESCRIPTION
**Pull request by JTY009**

### CODE REVIEW

File: test/support/CallOpsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ */
+
+package support
+
+import org.scalatest.{Matchers, WordSpec}
+
+class CallOpsSpec extends WordSpec with Matchers {
+
+  "addParamsToUrl" should {
+    val url = "http://localhost:1/"
```
Comment (arkowalski) [inline test/support/CallOpsSpec.scala]: 1?

File: conf/application.conf [hunk 2]
```diff
@@ -86,9 +86,6 @@ external-url {
   company-auth {
     host = "http://localhost:9949"
   }
-  tax-history-frontend {
```
Comment (johnrevill) [inline conf/application.conf]: This is still in use in the ConfigDecorator (https://github.com/hmrc/tax-history-frontend/blob/5cc42550bf23664b4e9d96b978f3cd8c85c6453e/app/config/ConfigDecorator.scala#L32)

Comment (johnrevill) [reply conf/application.conf]: Though, the port looks wrong

Comment (johnrevill) [reply conf/application.conf]: ...and I don't see any references to taxHistoryFrontendHost in the project. Guess that can disappear.

Comment (JTY009) [reply conf/application.conf]: tests passes when i removed it.

**Pull request merged**